beaker-vue/src/components/BeakerCell.vue

```diff
@@ -126,28 +130,23 @@ function handleDrop(item) {
      * it needs to disable dragging when using the cursor to select text.
      * Could accept props of disallowed classes if we want to make it generic.
  **/
-function handleDragStart(event, item) {
+function handleDragStart(event: DragEvent) {
 
-    const selectingText = event.target.nodeName === '#text';
-    if (selectingText) {
-        event.preventDefault();
-        return;
+    // const paintTarget = (event.target as HTMLElement).parentElement;
+    var paintTarget: HTMLElement|null = (event.target as HTMLElement);
+    while (paintTarget !== null && !paintTarget.classList.contains("beaker-cell")) {
```

Comment:
Is this needed because the `dragStart` is added to BeakerCell instead of the `DraggableMarker`?

Reply:
Additionally, is this just trying to find the closest `beaker-cell` parent?

If that's the case, I've used `closest` for that on `handleDrop`:

> target.closest('.drag-sort-enable');

Would that work here?

Reply:
Oh, I want to use closest, you're right. I thought this function existed, but forgot what it was called, and google failed me. But it was like a 2 line thing, so... But I'll fix it to use closest().

This doesn't have anything to do with changing where the dragging starts. This is how we are figuring out what to show when dragging.
By default, it will be the thing with the draggable tag, aka the event target. But we want to disply the whole cell when dragging, so we go up to find the cell that contains the target being dragged.
If all the cell elements had distinct ids, we could have replaced that with `paintTarget = document.getElementById()`